README.rst

```diff
@@ -78,89 +78,101 @@ It is also available online at http://docs.plone.org/develop/addons/bobtemplates
 Installation
 ------------
 
-Use in a buildout
-^^^^^^^^^^^^^^^^^
+Python Package Managers
+^^^^^^^^^^^^^^^^^^^^^^^
 
-::
+.. code-block:: console
 
-    [buildout]
-    parts += mrbob
+    # use a virtual environment for installation
 
-    [mrbob]
-    recipe = zc.recipe.egg
-    eggs =
-        mr.bob
-        bobtemplates.plone
+    # For Python 2 and before Python 3.4 use:
+    $ virtualenv <path_venv>
 
+    # For Python 3.4 and above use:
+    $ python -m venv <path_venv>
 
-This creates a mrbob-executable in your bin-directory.
-Call it from the ``src``-directory of your Plone project like this.::
+    # Activate your virtual environment
+    $ source <path_venv>/bin/activate
 
-    ../bin/mrbob -O collective.foo bobtemplates:plone_addon
+    # Install bobtemplates.plone
 
-Or to create a new theme package::
+    # via pip - prefered way
+    $ pip install bobtemplates.plone
 
-    mrbob -O plonetheme.tango bobtemplates:plone_theme_package
-    cd plonetheme.tango/
-    pip install -r requirements.txt
-    buildout bootstrap
-    bin/buildout
+    # via easy_install - old way if pip is not avaliable
```

Comment:
What would be that case actually? We could link to pip documentation on how to install it if it's not available, but I would refrain of giving alternative routes...